packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/toplevel.tsp

```diff
@@ -211,4 +220,19 @@ interface TopLevelTrackedResources {
     ```
     """)
   listBySubscription is ArmListBySubscription<TopLevelTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+      Resource notify operation.
+      Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.Resources/topLevelTrackedResources/top/actionSync
+      Expected query parameter: api-version=2023-12-01-preview
+      Expected request body:
+      ```json
+      {
+        "message": "Send a operation to top level.",
```

Comment:
nit, message better be "Resource action at top level."

Reply:
Fix message content in the new version.